drivers/SmartThings/zwave-lock/fingerprints.yml

```diff
@@ -288,6 +288,12 @@ zwaveManufacturer:
     productType: 0x0005
     productId: 0x0001
     deviceProfileName: lock-battery
+  - id: "TLJ"
```

Comment:
Please move this to `line 248` so that it is grouped with the other `base-lock` devices

add a comment with the manufacturer name above the fingerprint just like it is done elsewhere throughout the file.
```
  # TLJ
  - id: "TLJ"
```